### PR TITLE
MAISTRA-929: Updated resources to allow deployment on default OCP 4.2…

### PR DESCRIPTION
**Pull request by knrc**
… cluster

### CODE REVIEW

**Review by dmitri-d — APPROVED** [review]

File: deploy/smcp-templates/base [hunk 1]
```diff
@@ -4,26 +4,68 @@ spec:
...
           requests:
-            cpu: 100m
-            memory: 1G
+            cpu: 50m
+            memory: 128Mi
           limits:
-            cpu: 500m
-            memory: 4G
+            cpu: 50m
+            memory: 256Mi
```
Comment (brian-avery) [inline deploy/smcp-templates/base:68]: In the Jira, you tested 512mb memory and 100m CPU for Mixer, but it says 256mb and 50m here. Is this correct?

File: deploy/smcp-templates/base [hunk 2]
```diff
@@ -4,26 +4,68 @@ spec:
...
+        requests:
+          cpu: 100m
+          memory: 128Mi
+        limits:
+          cpu: 100m
+          memory: 256Mi
     pilot:
+      resources:
+        requests:
+          cpu: 100m
```
Comment (brian-avery) [inline deploy/smcp-templates/base:52]: Tested CPU from the Jira was 150m

Comment (knrc) [reply deploy/smcp-templates/base:52]: pilot pod is composed of two containers
- istio-proxy (50m, 50m, 128Mi, 256Mi)
- discovery (100m, 100m, 128Mi, 256Mi)

Combined these give pod resources of 150m, 150m, 256Mi, 512Mi

File: deploy/smcp-templates/base (hunk 1 above)
Comment (knrc) [reply deploy/smcp-templates/base:68]: telemetry pod is combined of two containers
- istio-proxy (50m, 50m, 128Mi, 256Mi)
- mixer (50m, 50m, 128Mi, 256Mi)

Combined these give pod resources of 100m, 100m, 256Mi, 512Mi

**Review by brian-avery — APPROVED** [review]

**Pull request merged**